src/components/swapv2/PairSuggestion/request.ts

```diff
@@ -0,0 +1,56 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+import Axios from 'axios'
+const SUGGEST_PAIR_API = process.env.REACT_APP_TYPE_AND_SWAP_URL || ''
```

Comment:
I think we don't need the fallback `''` here

Reply:
The request will still fail if it's `undefined`

Reply:
just want to pass typesript eslint.

Reply:
I think we should fallback by dev url

Reply:
removed `''`. there is no typescript error left.   

because in the past, I write fake api, it cauch eslint error, maybe.